src/main/java/cs/ualberta/cmput402/tictactoe/TicTacToeGame.java

```diff
@@ -31,7 +32,7 @@ public void promptNextPlayer(){
 
     public void playGame(){
         Scanner keyboardScanner = new Scanner(System.in);
-
+        board.initBoard();
```

Comment:
It's better to create a new instance of the board class and use the constructor of the class to initialize all variables whenever you want to start a new game instead of calling initBoard() method. This way you can make sure that all variables are initialized correctly and also makes the extension of the class easier. For example, you forgot to initialize the current player at the end of each round.
```suggestion
        board= new Board();
```

Reply:
I forgot to initialize the currentPlayer variable. Thanks for mentioning. 
I will edit this part.